src/sidebar.cls.php

```diff
@@ -0,0 +1,174 @@
+<?php
+/**
+ * The Sidebar class
+ *
+ * @since
+ * @package     LiteSpeed
+ * @subpackage  LiteSpeed/inc
+ * @author      LiteSpeed Technologies <info@litespeedtech.com>
+ */
+namespace LiteSpeed;
+
+defined( 'WPINC' ) || exit;
+
+class Sidebar extends Base {
+
+  /**
```

Comment:
Your file is using space not tab as indent.

Reply:
got it